recastai/apis/build/build.py

```diff
@@ -26,19 +26,27 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
   @token_required
-  def dialog(self, msg, conversation_id, language=None):
+  def dialog(self, msg, conversation_id, language=None, **options):
+
+    memory = {} if options.get('memory') is None else options.get('memory')
+    log_level = "info" if options.get('log_level') is None else options.get('log_level')
+    proxy = {} if options.get('proxy') is None else options.get('proxy')
+
     if language is None:
       language = self.language
 
-    params = {'message': msg, 'conversation_id': conversation_id}
+    params = {'message': msg, 'conversation_id': conversation_id, 'memory': memory, 'log_level': log_level, 'proxy': proxy}
```

Comment:
memory should only be sent if it's given as argument, otherwise the memory will be erased :) So you could remove `memory = {} if options.get('memory') is None else options.get('memory')` and instead add this line :
```python
if options.get('memory'):
  params['memory'] = options['memory']
```

Also, we should not send the proxy to Recast :)